bentoml/_internal/types.py

```diff
@@ -217,7 +217,7 @@ def is_compatible_type(t1: AnyType, t2: AnyType) -> bool:
 
 @json_serializer(fields=["uri", "name"], compat=True)
 @dataclass(frozen=False)
-class FileLike(t.Generic[t.AnyStr]):
+class FileLike(t.Generic[t.AnyStr], io.IOBase, t.IO[t.AnyStr]):  # type: ignore (python IO types)
```

Comment:
Is this necessary for runtime or just for typing?

Reply:
Just typing. I actually realized I can use `t.cast` later so I don't think this is necessary at all anymore.